src/studio/src/designer/frontend/dashboard/common/components/RepoList.tsx

```diff
@@ -225,6 +243,13 @@ export const RepoList = ({
               label={getLanguageFromKey('dashboard.open_in_new', language)}
             />,
           ];
+
+          if (isDatamodelling) {
```

Comment:
The last item should also be removed when it is a datamodelling repo, as it is essentially the same link, except it opens in a new tab by default

Reply:
Decided on just displaying the repos link as all other is not relevant:
![image](https://user-images.githubusercontent.com/10416981/145583141-265a2aef-2723-4b28-97a0-a3118b8688f3.png)

Maybe weird logic, but arr.splice returns the deleted item